### PR TITLE
refactor(Apply): FRMW-410 convert rxSelectFilter.Apply to quark

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-410

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxSelectFilter.Apply` to Quark filter.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Apply` Quark Filter

### Demo Page
screen shot 2015-12-18 at 4 25 05 pm

### API Docs
screen shot 2015-12-18 at 4 25 26 pm

**Comment by glynnis** [comment]
Design LGTM

File: src/quarks/Apply/docs/examples/Apply.simple.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            <rx-select-filter filter="filter"></rx-select-filter>
+          </rx-form-section>
+        </form>
+      </th>
+      <tr>
+        <th>Account</th>
+        <th>Status</th>
+        <th>Description</th>
+      </tr>
+  </thead>
```
Comment (CITguy) [inline src/quarks/Apply/docs/examples/Apply.simple.html]: Please correct indentation. `</thead>` through `</tbody>` are at incorrect indentation.

File: src/components/rxSelectFilter/rxSelectFilter.module.js [hunk 2]
```diff
@@ -10,13 +10,11 @@
...
  * * {@link rxSelectFilter.directive:rxSelectFilter rxSelectFilter}
  *
- * ## Filters
- * * {@link rxSelectFilter.filter:Apply Apply}
- *
  * ## Services
  * * {@link rxSelectFilter.service:SelectFilter SelectFilter}
  */
 angular.module('encore.ui.rxSelectFilter', [
+	'encore.ui.quarks',
```
Comment (CITguy) [inline src/components/rxSelectFilter/rxSelectFilter.module.js]: Please correct indentation.

File: src/quarks/Apply/docs/Apply.docs.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<p>
+  Used to apply an instance of <code>SelectFilter</code> to an array.
```
Comment (CITguy) [inline src/quarks/Apply/docs/Apply.docs.html]: Considering that this Quark references the `SelectFilter` quark (to be merged via #1478), please link to the `SelectFilter` quark in both the demo and the jsdocs.

_Note:_ this PR will need merged *after* #1478

**Comment by 100stacks** [comment]
@CITguy comments implemented. :+1:

File: src/quarks/quarks.module.js [hunk 4]
```diff
@@ -16,6 +16,7 @@
  * * {@link quarks.filter:titleize titleize}
  * * {@link quarks.filter:rxEnvironmentMatch rxEnvironmentMatch}
  * * {@link quarks.filter:rxEnvironmentUrl rxEnvironmentUrl}
+ * * {@link quarks.filter:Apply Apply}
```
Comment (CITguy) [inline src/quarks/quarks.module.js]: Please remove the change to this file. We'll revisit updating this file once all Quarks have been migrated.

**Comment by 100stacks** [comment]
@CITguy okay reverted updates to `quarks.module.js` and rebased. :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Comment by halkon** [comment]
LGTM

**Pull request merged**